.travis-ci.yml

```diff
@@ -0,0 +1,10 @@
+language: minimal
+
+services:
+  - docker
+
+install:
+    - travis_retry docker-compose build
```

Comment:
could delegate to `make build` for consistency?

Reply:
- updated `make build` to build all images.
- updated `.travis-ci.yml` to use `make build`.